examples/apigee/v1beta1/instanceattachment.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: apigee.gcp.upbound.io/v1beta2
+kind: Organization
```

Comment:
Since this is the YAML file of the `InstanceAttachment` resource, it would be better if you put the `InstanceAttachment` resource at the beginning of the file.

Reply:
done